<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.196629 rgba(66, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ec">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MingLiU-ExtB</family>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>NoAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(75, 240, 255);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.303371, y1:0.295, x2:1, y2:1, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_star">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(75, 240, 255);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.303371, y1:0.295, x2:1, y2:1, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>oct</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rightskob">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_leftskob">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid white;
  background-color: rgb(142, 249, 255);
}
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(76, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(75, 240, 255);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.303371, y1:0.295, x2:1, y2:1, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_procent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(75, 240, 255);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.303371, y1:0.295, x2:1, y2:1, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>hex</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusminus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="resualt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
qproperty-alignment:  'AlignRight | AlignCenter';
  border: 1px solid white;
background-color : qconicalgradient(cx:0, cy:1, angle:98.2, stop:0.814607 rgba(140, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>41</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(75, 240, 255);
  color: white; 
  border: 1px solid white;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.303371, y1:0.295, x2:1, y2:1, stop:0 rgba(0, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>bin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(142, 255, 255);
  border: 1px solid white;
}
QPushButton:pressed {
    background-color:qlineargradient(spread:pad, x1:0.567416, y1:0.148, x2:1, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_ec</zorder>
   <zorder>pushButton_star</zorder>
   <zorder>pushButton_sin</zorder>
   <zorder>pushButton_divide</zorder>
   <zorder>pushButton_rightskob</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_leftskob</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_AC</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_procent</zorder>
   <zorder>pushButton_cos</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_plusminus</zorder>
   <zorder>resualt</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_dot</zorder>
   <zorder>pushButton_bin</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
